### ISSUE TITLE
Empty result when searching by _id in a custom schema table

### ISSUE THREAD

**Issue by krzaczek** [crate/crate#1815]
Crate.io version 0.47.7

```sql
create table my_table (
first_column integer,
 second_column string
);

insert into my_table (first_column, second_column)
values ('123', '456');

select _id from my_table limit 100;
```
Returns: _id: AUw3V8zMTVvl9Ie6E1eS

```sql
select * from my_table where _id = 'AUw3V8zMTVvl9Ie6E1eS' limit 100;
```
Returns our record.

Same scenario but with a custom schema

```sql

create table testing.my_table (
first_column integer,
 second_column string
);

insert into testing.my_table (first_column, second_column)
values ('123', '456');

select _id from testing.my_table limit 100;
```
Returns _id: AUw3XWDnTVvl9Ie6E1eT

But if we try searching by _id field we get an empty result
```sql
select * from testing.my_table where _id = 'AUw3XWDnTVvl9Ie6E1eT' limit 100;
```
Same thing if we set a primary key field;

```sql
create table testing.my_table1 (
  id integer primary key,
first_column integer,
 second_column string
);

insert into testing.my_table1 (id, first_column, second_column)
values (1, '123', '456');

select * from testing.my_table1 where id = 1 limit 100;
```
Returns empty result.

**Comment by krzaczek** [comment]
Did a test on play.crate.io and version 0.47.6 returns an error:

select * from testing.my_table where _id = 'AUw3XWDnTVvl9Ie6E1eT' limit 100;
SQLActionException[Table 'my_table' unknown].

Looks like "schema" is ignored.

**Comment by seut** [comment]
thanks for reporting we'll take care of this asap. fyi, this issue does not occur on the 0.48 testing release.

**Comment by krzaczek** [comment]
Ok thx for the info will check 0.48.0 

pawel (at) mobile

**Comment by mfelsche** [comment]
fixed by #1818

thanks for reporting!

**Comment by mfelsche** [comment]
will be released with the next 0.47 hotfix release (0.47.8)
